src/qibolab/dummy/platform.py

```diff
@@ -1,7 +1,7 @@
 import itertools
 import pathlib
 
-from qibolab.channels import Channel, ChannelMap
+from qibolab.instruments.channels import Channel, ChannelMap
```

Comment:
Are there `channels` under `instruments`? I don't think so. Maybe
```suggestion
from qibolab.channel import Channel, ChannelMap
```
(just noticed that is duplicate of https://github.com/qiboteam/qibolab/pull/885/files#r1580801701 )